internal/services/monitor/monitor_activity_log_alert_resource.go

```diff
@@ -57,6 +58,8 @@ func resourceMonitorActivityLogAlert() *pluginsdk.Resource {
 
 			"resource_group_name": commonschema.ResourceGroupName(),
 
+			"location": commonschema.LocationOptional(),
```

Comment:
Apologies, I think we want to set the default as `Global` here like you had it previously, however this property should probably become required in the next major release, so can we make this 
```suggestion
			"location": func() *pluginsdk.Schema {
				if !features.FourPointOhBeta() {
					return &pluginsdk.Schema{
						Type:             schema.TypeString,
						Optional:         true,
						Default:          "global",
						ForceNew:         true,
						StateFunc:        location.StateFunc,
						DiffSuppressFunc: location.DiffSuppressFunc,
					}
				}
				return commonschema.Location()
			}(),
```